src/NzbDrone.Core/Notifications/Telegram/TelegramSettings.cs

```diff
@@ -32,6 +40,12 @@ public class TelegramSettings : IProviderConfig
         [FieldDefinition(3, Label = "NotificationsTelegramSettingsSendSilently", Type = FieldType.Checkbox, HelpText = "NotificationsTelegramSettingsSendSilentlyHelpText")]
         public bool SendSilently { get; set; }
 
+        [DefaultValue(true)]
+        [FieldDefinition(4, Label = "NotificationsTelegramSettingsSendMetadataLink", Type = FieldType.Checkbox, HelpText = "NotificationsTelegramSettingsSendMetadataLink")]
+        public bool SendMetadataLink { get; set; }
+        [FieldDefinition(5, Label = "NotificationsTelegramSettingsMetadataLinkType", Type = FieldType.Select, SelectOptions = typeof(MetadataLinkType), HelpText = "NotificationsTelegramSettingsMetadataLinkType")]
```

Comment:
`TelegramSettingsValidator` needs a validation rule to ensure the value of the enum is valid.

An example how that's done for `ReleaseType`:
```
RuleFor(c => c.Value).Custom((releaseType, context) =>
{
    if (!Enum.IsDefined(typeof(ReleaseType), releaseType))
    {
        context.AddFailure($"Invalid release type condition value: {releaseType}");
    }
});
```